src/Microsoft.CodeAnalysis.Analyzers/Core/FixAnalyzers/FixerWithFixAllAnalyzer.cs

```diff
@@ -29,6 +29,7 @@ public sealed class FixerWithFixAllAnalyzer : DiagnosticAnalyzer
         private const string EquivalenceKeyPropertyName = "EquivalenceKey";
         private const string EquivalenceKeyParameterName = "equivalenceKey";
         internal const string CodeFixProviderMetadataName = "Microsoft.CodeAnalysis.CodeFixes.CodeFixProvider";
+        private const string ExportCodeFixProviderAttributeMetadataName = "Microsoft.CodeAnalysis.CodeFixes.ExportCodeFixProviderAttribute";
```

Comment:
Do we have access to WellKnownTypeNames from this location?

Reply:
Correct, this definition should move to https://github.com/dotnet/roslyn-analyzers/blob/main/src/Utilities/Compiler/WellKnownTypeNames.cs

Reply:
Just out of interest, what's the criteria for defining a type in `WellKnownTypeNames`? Does it have to be used in multiple places, or should all type names be defined there? Should `CodeFixProvider` be defined there as well?

Reply:
All strings representing fully qualified name to some type should be defined there I think. So, `CodeFixProvider` should also be there, I think.